CHANGES.md

```diff
@@ -3,6 +3,7 @@
 - [IMPROVED] Added HTTP status code to `Response` objects.
 - [FIX] Too many bytes written exception caused by inconsistent encoding between UTF-8 and the
   JVM default. UTF-8 is now correctly used for the request body content length and throughout.
+- [FIX] Made cookie authentication work when using custom SSL configurations.
```

Comment:
I think this should be reworded, to something along the lines of 
-[FIX] `CookieInterceptor` now honours Custom SSL Configurations 

or similar